phpBB/phpbb/extension/extension_interface.php

```diff
@@ -27,6 +27,13 @@
 	public function is_enableable();
 
 	/**
+	* Indicate why the extension can not be enabled.
+	*
+	* @return string|array with actual processed language string(s) (not language keys), or empty
+	*/
+	public function not_enableable_reason();
```

Comment:
A such method does not belong in the language manager. There is too few usage and to few logic. Actually there is nothing inside, just a loop.

About the solution with 2 methods, as I said many time, I don't like and I won't validate it myself.